pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1095,7 +1095,7 @@ func (dm *DockerManager) pluginDisablesDockerNetworking() bool {
 
 // newDockerVersion returns a semantically versioned docker version value
```

Comment:
Which brings up a possible issue..?  Version() and APIVersion() returning the same value.